buildSrc/build.gradle.kts

```diff
@@ -12,6 +12,16 @@ repositories {
   google()
   gradlePluginPortal()
   mavenCentral()
+  maven("https://maven.pkg.jetbrains.space/public/p/kotlinx-html/maven") {
```

Comment:
Could you say what we are using this for? It looks exciting.

Reply:
It's for [Dokka](https://github.com/Kotlin/dokka/releases/tag/v1.4.32), I'll add a comment.